ReactSkia/components/RSkComponentText.cpp

```diff
@@ -42,6 +42,27 @@ inline SkScalar yPosOffset(AttributedString attributedString, SkScalar paraHeigh
     return 0;
 }
 
+inline void ellipSize(std::shared_ptr<ParagraphBuilder> paraBuilder, std::vector<LineMetrics>& metrics, ParagraphAttributes paragraphAttributes, Rect frame) {
+    int i=0;
+    SkScalar minNumberOfLines = 0;
+    ParagraphStyle paraStyle;
+
+    for(; i< metrics.size(); i++) {
+        if ((metrics[i]).fBaseline > frame.size.height) {
+            minNumberOfLines = (metrics[i]).fLineNumber -1;
+            break;
+        }
+    }
+
+    if ((minNumberOfLines > 0) && (minNumberOfLines < paragraphAttributes.maximumNumberOfLines))
+        paraStyle.setMaxLines(minNumberOfLines);    
+    else 
+        paraStyle.setMaxLines(paragraphAttributes.maximumNumberOfLines);
+
```

Comment:
we have to set maxLines and ellipsis only if "minNumberOfLines" and "paragraphAttributes.maximumNumberOfLines" is greater than 0 .
As per https://reactnative.dev/docs/0.64/text#numberoflines

Reply:
updated